packages/dynamite/dynamite/lib/src/models/openapi/schema.dart

```diff
@@ -61,6 +61,10 @@ abstract class Schema implements Built<Schema, SchemaBuilder> {
 
   int? get maxLength;
 
+  int? get minItems;
+
+  int? get maxItems;
+
```

Comment:
This and other validators should be documented in the doc comment.
Probably something for a followup so please open an issue.

Reply:
Aren't you doing that in https://github.com/nextcloud/neon/pull/1931 already?

Reply:
nope

Reply:
I meant that the constraints of a parameter should be added to the doc comments of the generated method/attribute. i.e that the generated code has these documentations.

Not that the dynamite code has them. You are right that I did add thorough comments to them.

Reply:
https://github.com/nextcloud/neon/issues/1944